cmd/module.go

```diff
@@ -47,9 +54,37 @@ func init() {
 		"The name of the module")
 	buildModuleCmd.Flags().StringVarP(&buildModuleCmdPlatform, "platform", "p", "cf",
 		`The deployment platform; supported platforms: "cf", "xsa", "neo"`)
+
+	// sets the flags of the solo Makefile command build module
+	soloBuildModuleCmd.Flags().StringVarP(&soloBuildModuleCmdSrc, "source", "s", "",
+		"The path to the MTA project; the current path is set as default")
+	soloBuildModuleCmd.Flags().StringVarP(&soloBuildModuleCmdTrg, "target", "t", "",
+		"The path to the folder in which the module build results are created; the <current folder>/.<project name>_mta_build_tmp/<module name> path is set as default")
```

Comment:
1. It has different name. We don't want to change original one because we don't want to break existing makefiles of the client.
2. Platform flag is not relevant for this command. We'll always package the module.

Reply:
I meant specifically the description for the `target` parameter

Reply:
yes, it's different. Description confirmed by Paola and Natalia.

Reply:
This part: `the <current folder>/.<project name>_mta_build_tmp/<module name> path is set as default` should be copied to `mbt module build` I think, since the temporary build results aren't saved in the current path like it currently says

Reply:
changed